app-frontend/src/app/pages/projects/edit/annotate/annotate.scss

```diff
@@ -79,19 +79,37 @@
 }
 
 .annotate-clone-marker {
-    background: rgba(255, 68, 51, 0.5);
+    background: rgba(229, 115, 115, 0.5);
```

Comment:
I'd create a new color variable for this in `_colors`, then modify the transparency of it for the separate parts
https://robots.thoughtbot.com/controlling-color-with-sass-color-functions